vocabularies/gsq-features.ttl

```diff
@@ -0,0 +1,78 @@
+        gafeat:tectonicStructuralFrameworkFeature,
+        gafeat:subBlock ;
+    skos:prefLabel "GSQ Geoscience and Administrative Features"@en .
+
+gafeat:block a skos:Concept ;
+    skos:definition "A geographic grid used for exploration tenure management using the basic unit (a BLOCK) defined as five minutes of latitude by five minutes of longitude. The Blocks are grouped by 1:1 000 000 sheet areas (the basic map unit for description) referred to as Block Identification Map (BIM). Each BIM contains 3456 blocks with the Blocks numbered between 1 and 3456 inclusive."@en ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Block"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:lithodemicFeature a skos:Concept ;
+    skos:definition "A lithodemic feature is a three-dimensional, non-stratified body of rock, comprising one or more intrusive, highly-deformed and/or metamorphosed rock types, to which the Law of Superposition is not applicable, although age is an important criterion; contacts with other units may be intrusive, extrusive, tectonic, metamorphic and/or sedimentary."@en ;
+    dct:source "NACSN, 1983, 2005; Kumpulainen, R.A., 2017." ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Lithodemic feature"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:permit a skos:Concept ;
+    skos:definition "In general mineral or energy rights in Australia are reserved to the Crown. The acquisition of rights to minerals or energy stems from legislative frameworks in Queensland. These frameworks provide initially for exploration of the resource, and consist of the grant by the Crown in the form of exploration permits, licences or leases. Exploration permits, leases or licences permit works to be undertaken to determine the likely existence of minerals or resources. Actual mining is subject to a further grant of mining or minerals production leases or licences. The legislation also provides for the payment of royalties to the State and to compensate the owners or occupiers of the surface land."@en ;
+    skos:example "ATP1234, ML5678, EPM9876"@en ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Permit"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:placeName a skos:Concept ;
+    skos:definition "Place Names are names given to natural and artificial geospatial features, such as administrative areas, political regions, mountain ranges, rivers, bays etc."@en ;
+    skos:inScheme gafeat:conceptScheme ;
+    skos:prefLabel "Place name"@en ;
+    skos:topConceptOf gafeat:conceptScheme .
+
+gafeat:resourceAccumulation a skos:Concept ;
+    skos:definition "Resource accumulations are naturally occurring accumulations or concentrations of metals, minerals or hydrocarbons of sufficient size and concentration that might, under favourable circumstances, have economic value. Economic concentrations of metals or other mineral commodities are known as ore."@en ;
```

Comment:
"metals, minerals or hydrocarbons".
Should we include quarry resources here too? Is coal implicit in minerals?

"Resource accumulations are naturally occurring accumulations or concentrations of metals, minerals, coal, hydrocarbons, or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore."

Reply:
What about the following?

"Resource accumulations are naturally occurring accumulations or concentrations of minerals (including metals and coal), hydrocarbons, or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore."

Reply:
Forgot about quarrying:

"Resource accumulations are naturally occurring accumulations or concentrations of minerals (including metals and coal), hydrocarbons, quarrying resources or other geological material that is of sufficient size and concentration that, under favourable circumstances, may have economic value. Economic concentrations of metals or other mineral commodities are known as ore.

Reply:
I like the definition by @johnmckellar